examples/wifi/wifi_aware/nan_console/main/nan_main.c

```diff
@@ -103,7 +103,7 @@ static struct {
 } ping_args;
 
 static const char *TAG = "nan_console";
-static esp_netif_t *g_nan_netif;
+static esp_netif_t *g_nan_netif = NULL;
```

Comment:
This is redundant, but might as well.

Reply:
In fact, this is not redundant. When the variable is declared without value, its behavior is unpredictable. This is because the allocated memory address may contain “garbage”.  Also, declaring static variable with value is a MISRA C rule.

Reply:
@entrop1a , since it's a **global** variable, it'll be initialised to NULL

Reply:
Understood. I will fix it.